otherlibs/unix/fsync.c

```diff
@@ -0,0 +1,28 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           */
```

Comment:
This copyright header is wrong!

Reply:
Where can I get a correct one then?

Reply:
Your name isn’t Xavier, you’re not at Inria, and I hope you haven’t been sitting on the patch for 22 years? 🙂